libs/langchain/langchain/document_loaders/parsers/pdf_docai.py

```diff
@@ -0,0 +1,273 @@
+"""Module contains a PDF parser based on DocAI from Google Cloud.
```

Comment:
Is this just a PDF parser or does it parse other types of files? 

if it's only doing pdf, then prior to merging, I or another maintainer will create a `pdf` namespace to place this parser under. 

If it's not only pdf, we should change the module name to docai

Reply:
good point, renamed to docai.